README.md

```diff
@@ -1,6 +1,24 @@
+[![License](https://img.shields.io/github/license/dolbyio-samples/blog-nextjs-edu-platform-starter)](LICENSE)
+
+![Blog Picture](https://dolby.io/wp-content/uploads/2021/09/How-to-Add-Quality-Assurance-to-Educational-Video-Production-with-Nextjs-1.jpg)
+
 # Dolby.io Education Platform - Next.js Starter
+This repository [complements the blog](https://dolby.io/blog/how-to-add-quality-assurance-to-educational-video-production-with-next-js/) about integration of Dolby.io's API into an educational video platform built with Next.js.
+
+## Overview
+To build quality assurance into educational content, Next.js can be leveraged for integrating aspects such as video processing, content validation, test automation, and user engagements. This helps offer insightful and practical examples to help developers enhance the quality and reliability of their educational videos. 
+
+## Requirements
+To follow along with the tutorial, you will need: 
+- Basic knowledge of JavaScript
+- Familiarity with [Next.js](https://nextjs.org/docs#system-requirements)
+- Node.js 16.8+ and npm installed
+- An S3 bucket or other storage provider that contains educational videos
+- A [Dolby.io account](https://dashboard.dolby.io/signup/)
 
-## Installation
+Here we will be using the Media APIs, specifically the Diagnose API, therefore you would need [an API key](https://dolby.io/developers/media-processing/quick-start/analyzing-media#1-get-your-api-key). You receive 50GBs to start out for free, which should help you get started with this project. 
```

Comment:
Replace the URL to https://docs.dolby.io/media-apis/docs/quick-start-to-analyzing-media#1-get-your-api-token